### ISSUE TITLE
Missing alt text fields

### ISSUE THREAD

**Issue by joemull** [openlibhums/janeway#5453]
**Describe the bug**
I found a few alt text bugs when extending the feature to Hourglass.

There is no way to set alt text for these images:
  - Account.profile_image
  - Press.secondary_image
  - NewsItem.large_image_file
  - Press.default_carousel_image

This property does not exist (called in several templates):
  - `news_item.best_image_alt_text`

**Janeway version**
1.9 / 4e0c64c447e04843ab1232ddd99e983dfedd8bc8

**Comment by ajrbyers** [comment]
BARF Note:

- For profile images: allow staff and editors to set alt text

**Comment by mauromsl** [comment]
closed by #5520
